Fix DITA warnings in Tuning Performance guide (#4863)
theforeman/foreman-documentation: guides/common/modules/con_hardware-and-operating-system-configuration.adoc

Network::
The communication between the {ProjectServer} and {SmartProxies} is impacted by the network performance.
A decent network with a minimum jitter and low latency is required to enable hassle free operations such as {ProjectServer} and {SmartProxies} synchronization.

> At the very leas, ensure it is not causing connection resets and similar problems.

Reviewer: ```suggestion
At the very least, ensure it is not causing connection resets and similar problems.
```

typo

Network::
The communication between the {ProjectServer} and {SmartProxies} is impacted by the network performance.

> A decent network with a minimum jitter and low latency is required to enable hassle free operations such as {ProjectServer} and {SmartProxies} synchronization (at least ensure it is not causing connection resets, and so on).

Reviewer 2: Perhaps we should try to avoid using parentheses altogether? I think that's  defined somewhere in the IBM guide.

```suggestion
A decent network with a minimum jitter and low latency is required to enable hassle free operations such as {ProjectServer} and {SmartProxies} synchronization.
At the very leas, ensure it is not causing connection resets and similar problems.
```

Reviewer 3: Correct. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses
